setup.sh

```diff
@@ -38,30 +41,39 @@ while getopts "dhn:" opt; do
 	esac
 done
 
-# Setup the LCG environment
-$ECHO "Getting the LCG environment ... "
-source $LCG/setup.sh
+# Setup the base environment
+if [[ "$useLCG" == "1" ]]; then
+        $ECHO "\nGetting the LCG environment ... "
+        source $LCG/setup.sh
+else
+        $ECHO "\nAssuming you are in the right Singularity container ... "    
```

Comment:
You can actually check the environment variable `$SINGULARITY_CONTAINER`

Reply:
Nice using it now